chalice/app.py

```diff
@@ -49,14 +49,14 @@ def dispatch(*args, **kwargs):
     for route, methods in routes.items():
         app.route(route, methods=list(set(methods) - {"OPTIONS"}), cors=True)(dispatch)
 
-    # with open(os.path.join(pkg_root, "index.html")) as fh:
-    #     swagger_ui_html = fh.read()
-    #
-    # @app.route("/")
-    # def serve_swagger_ui():
-    #     return chalice.Response(status_code=200,
-    #                             headers={"Content-Type": "text/html"},
-    #                             body=swagger_ui_html)
+    with open(os.path.join(pkg_root, "index.html")) as fh:
+        swagger_ui_html = fh.read()
```

Comment:
Does chalice make the "index.html"?

Reply:
No, I looked at Upload's `index.html` and it's a template with a bunch of CSS that populates the content by reading in a Swagger yaml, so I copied it over.

Reply:
I would think that Swagger has a tool to generate the `index.html`, but I don't know where/what that is